openedx/features/wikimedia_features/meta_translations/apps.py

```diff
@@ -17,3 +17,6 @@ class MetaTranslationsConfig(AppConfig):
             }
         }
     }
+
+    def ready(self):
+        from . import signals  # pylint: disable=unused-import
```

Comment:
why do we need this?

Reply:
Signals file contains few signals to track new changes so that only updated data related API(s) can be hit on send_translation cron job.

Details usecases can be view from **page-5** of following link:
https://drive.google.com/file/d/149nbsLtfrn2OrAvlgirOSUh-lDU2-MPe/view?usp=sharing